### ISSUE TITLE
Move sources to src folder.

### ISSUE THREAD

**Issue by mforbes** [sphinx-contrib/cookiecutter#22]
I recommend moving the package to `src/sphinxcontrib/` so that tests don't accidentally test against the source.

(I ran into an issue when switching package formats: my tests all passed, but the package was not actually getting installed!)

**Comment by stephenfin** [comment]
Good idea. Would you be able to a provide a PR to do this? I'd be happy to review/merge it.
